### PR TITLE
Simperium Sunset Mark I: Data Model

### PR DESCRIPTION
**Pull request by jleandroperez**
### Details:
This PR lays the groundwork required to deintegrate Simperium. Reference #6086.

- Added Model Mark 53
 - Updated `Notification` Entity
 - Nuked `Meta` Entity
- Added Model 52 > 53 Migration Test
- Fixed few Unit Tests

### Testing:
- Verify that the app builds and runs correctly
- Verify that the Unit Tests run as expected
- Note that Notifications Sync'ing is *expected* to be broken here. This PR is mapped to a midway branch, so we're good

Needs Review: @astralbodies 
Aaron, may i bug you with a fun CoreData-Y PR?

Thanks in advance!

cc @kwonye

### CODE REVIEW

**Comment by jleandroperez** [comment]
@0nko apologies about the noise, it would appear GitHub went crazy!

File: WordPress/Classes/System/WordPressAppDelegate.m [hunk 1]
```diff
@@ -751,6 +751,10 @@ - (void)loginSimperium
 
 - (void)logoutSimperiumAndResetNotifications
 {
+    if (self.testSuiteIsRunning) {
```
Comment (astralbodies) [inline WordPress/Classes/System/WordPressAppDelegate.m]: I'm assuming this is just temporary until Mark 2+

Comment (jleandroperez) [inline WordPress/Classes/System/WordPressAppDelegate.m]: Yes sir! this method will be entirely gone, just a temporary patch.

Thanks for checking!

**Comment by jleandroperez** [comment]
Thanks for the review sir!

**Pull request merged**